scheduler/scheduler.go

```diff
@@ -500,35 +500,9 @@ func (s *scheduler) startTask(id, source string) []serror.SnapError {
 		return errs
 	}
 
-	// Group dependencies by the node they live on
```

Comment:
**To reviewers**:
This code from in lines 504-531 occurred in few places, so I decided to put that in [(t *task) SubscribePlugins()](https://github.com/intelsdi-x/snap/pull/1548/files#diff-77f163e381dd113e90ebd1bd76479b54R364)